src/device-registry/config/constants.js

```diff
@@ -1,6 +1,7 @@
 const mongoose = require("mongoose");
 const ObjectId = mongoose.Types.ObjectId;
-const { generateDateFormatWithoutHrs } = require("../utils/date");
+const log4js = require("log4js");
+const logger = log4js.getLogger(`${this.ENVIRONMENT} -- constants-config`);
```

Comment:
Thanks @BenjaminSsempala 

Does `this.ENVIRONMENT` work? Have you tested it out?

Thanks!

Reply:
Yes @Baalmart , It seemed to work fine. 
Although when it seems some new circular dependencies have emerged on staging. Let me try to address those too.

Reply:
Alright, thanks! It is becoming a real nightmare indeed!